### DISCUSSION TITLE
Reporting api and dashboard

### DISCUSSION THREAD

**Discussion by SGFGOV** [medusajs/medusa#3367] [Feature Requests]
Can we have a reporting dashboard in the ad.in - like shopify does..I know there is feature Request for reporting api

**Comment by SGFGOV** [comment]
Is there any progress on this

**Comment by tsvetann** [comment]
@SGFGOV I think with the new admin architecture (coming soon) we will be able to create new pages and widgets in medusa admin which will open up possibilities to develop reporting dashboard plugins

**Reply by SGFGOV** [reply]
True
. But will need the apis ideally
